apps/src/templates/studioHomepages/JoinSection.jsx

```diff
@@ -7,9 +8,9 @@ const styles = {
   main: {
     borderWidth: 1,
     borderStyle: 'solid',
-    borderColor: color.charcoal,
-    height: 68,
-    width: 940,
+    borderColor: color.border_gray,
+    height: 72,
+    width: 970,
```

Comment:
Brad added a nice constant for this. Example usage: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/Notification.jsx#L23